### PR TITLE
fix: credits rounding on credit batch page

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: regen-network/regen-web#1834

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

https://deploy-preview-1843--regen-registry.netlify.app/credit-batches/C56-002-20140701-20141231-001
https://deploy-preview-1843--regen-registry.netlify.app/credit-batches/C55-002-20170101-20170131-001

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-registry/src/components/molecules/BatchTotalsGrid.tsx [hunk 1]
```diff
@@ -1,14 +1,11 @@
 import React from 'react';
 import { Grid, SxProps, Theme } from '@mui/material';
+import { quantityFormatNumberOptions } from 'config/decimals';
```
Comment (blushi) [inline web-registry/src/components/molecules/BatchTotalsGrid.tsx:3]: just using the formatting config that's already used throughout the whole app:
```
export const QUANTITY_MIN_DECIMALS = 0;
export const QUANTITY_MAX_DECIMALS = 6;

export const quantityFormatNumberOptions = {
  minimumFractionDigits: QUANTITY_MIN_DECIMALS,
  maximumFractionDigits: QUANTITY_MAX_DECIMALS,
};
```

cc/ @clevinson in the case of C56-002-20140701-20141231-001, I don't think that makes sense to display decimals when the value is already an integer, so I'm still using 0 for `minimumFractionDigits`

**Comment by blushi** [comment]
@erikalogie @clevinson see testing instructions above

**Review by flagrede — APPROVED** [review]

**Comment by blushi** [comment]
> @erikalogie @clevinson see testing instructions above

up

**Comment by erikalogie** [comment]
@blushi looks fine to me

**Review by clevinson — APPROVED** [review]
Lgtm. Thx @blushi

**Pull request merged**